File: Logic Building Problems/Basic Problems/Check Even or Odd/logic1.js
// A simple Javascript program to
// check for even or odd

function isEven(n) { 
    return (n % 2 == 0); 
}

// Driver code

    let n = 101;
    if (isEven(n)) {
        console.log("true");
    } else {
        console.log("false");
    }

File: Logic Building Problems/Basic Problems/Check Even or Odd/logic1.cpp
// A simple C++ program to check for even or odd
#include <iostream>
using namespace std;

bool isEven(int n) { 
  return (n % 2 == 0); 
}

int main() {
    int n = 101;
    if(isEven(n))
         cout << "true";
      else 
        cout << "false";

    return 0;
}